test/cmdlineTests.sh

```diff
@@ -378,6 +379,12 @@ SOLTMPDIR=$(mktemp -d)
         then
             opts+=(-o)
         fi
+
+        if grep "pragma solidity ^$developmentVersion\|pragma solidity >=$developmentVersion" "$f" >/dev/null
+        then
+            echo "pragma solidity >0.0.1;" > "temp.sol"
+            grep -v "pragma solidity ^$developmentVersion\|pragma solidity >=$developmentVersion" "$f" >> "temp.sol" && mv "temp.sol" "$f"
+        fi
```

Comment:
This would be simpler with sed:
```suggestion
        sed -i.bak -E -e 's/pragma solidity (\^|>=)'"$developmentVersion"'/pragma solidity >0.0.1/' "$f"
```

BTW, note that the dots inside `$developmentVersion` will be interpreted as in regexes (i.e. arbitrary character will match) rather than literal dots, both in your and in my version. For example `0.8.3` will match also `0.823`. Hopefully this will not happen in practice.

Reply:
```
sed -i.bak -E -e 's/pragma solidity(\^|>=)'"$developmentVersion"'/pragma solidity >0.0.1/' "$f"
```

Reply:
Looks like in `-E` mode it's `\s` that causes problems :)

`[[:space:]]` works instead on both macOS and Linux. So I guess something like this would work
```
sed -i.bak -E -e 's/pragma[[:space:]]+solidity[[:space:]]*(\^|>=)[[:space:]]*'"$developmentVersion"'/pragma solidity >0.0.1/' "$f"
```
It's a bit verbose for my taste though. Too bad `\s` does not work.

Reply:
I did the one that could handle more spaces.